src/index.html

```diff
@@ -20,8 +20,692 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="header">
+        <div class="header__box">
+          <a
+            class="header__logo"
+            href="index.html"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="header__img"
+            />
+          </a>
+          <nav class="nav">
+            <ul class="nav__menu">
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link is-active"
+                >
+                  Apple
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Samsung
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Smartphones
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Gadgets
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Tablets
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Photo
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <main class="main">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__img">
+            <img
+              src="images/imac.jpeg"
```

Comment:
Consider providing a more descriptive `alt` text for the product image to improve accessibility. Instead of 'image-Imac', use something like 'Apple iMac 27 Retina'.